src/python/grpcio/grpc/__init__.py

```diff
@@ -721,6 +768,26 @@ def peer(self):
     raise NotImplementedError()
 
   @abc.abstractmethod
+  def parent(self, child=None, census=None):
+    """Creates a value to associate invoked RPCs with the current RPC.
+
+    This method's parameters have no guaranteed defaults; if either True or
+    False is desired for the value of an argument the argument should be
+    explicitly passed.
+
+    Args:
+      child: An optional boolean indicating whether or not the invoked RPC
```

Comment:
Not only is it not valid, it's not _possible_ - try writing such client-side code?